.travis.yml

```diff
@@ -0,0 +1,22 @@
+matrix:
+  include:
+    - language: python
+      python:
+      - "3.6"
+      install:
+        - pip3 install -r requirements3.txt
+      script:
+        - make test
+        - make lint
+        - make mypy
+
+    - language: node_js
+      node_js:
+        - "8"
+      before_install:
+        - cd amundsen_application/static
+      install:
+        - npm install
+      script:
+        - npm run build
+        - npm run test
```

Comment:
Because our unit tests are still TBD, `npm run test` will fail. In addition, `npm run build` builds our source so I'm wondering if we want to run that on Travis. In the event we don't run `npm run build`, we'd instead want to run `npm run tsc` for the type checks. However for some reason `npm run tsc` fails locally so I assume it would fail on Travis as well -- I have a ticket somewhere to investigate.

What do you think of only configuring `npm run lint` for now? Later we can add `npm run test` when unit tests are completed/fixed.

Reply:
I think we need to either fix the test or disable this as it will block the deployment step that will be added shortly.